package.json

```diff
@@ -56,6 +56,7 @@
     "@guardian/src-ed-lines": "^3.6.0-rc.0",
     "@guardian/src-foundations": "^3.6.0",
     "@guardian/src-icons": "^3.7.0",
+    "@guardian/src-layout": "^3.8.0",
```

Comment:
I've added the version of `Hide` from Source here. DCR has it's own `Hide` but I've been meaning to add this shared one for _ages_, the api of @SiAdcock version is so much better than mine 😊